src/js/components/AccordionPanel.js

```diff
@@ -0,0 +1,74 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
+
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+import Box from './Box';
+import Heading from './Heading';
+import ListItem from './ListItem';
+import OpenIcon from './icons/base/Add';
+import CloseIcon from './icons/base/Subtract';
+import Collapsible from './Collapsible';
+
+import CSSClassnames from '../utils/CSSClassnames';
+
+const CLASS_ROOT = CSSClassnames.ACCORDION_PANEL;
+
+export default class AccordionPanel extends Component {
```

Comment:
Since this is a ListItem inside a List, perhaps it should be called AccordionListItem?